numbat-cli/src/completer.rs

```diff
@@ -6,7 +6,7 @@ use rustyline::completion::{extract_word, Completer, Pair};
 pub struct NumbatCompleter {
     pub context: Arc<Mutex<Context>>,
     pub modules: Vec<String>,
-    pub all_timezones: Vec<&'static str>,
+    pub all_timezones: Vec<String>,
```

Comment:
This is not at all critical for us, but it was nice to be able to get the timezones as references before. Now we need to copy them over.

Reply:
Yeah I think `chrono-tz` could do this because it exposed the fact that all time zones were embedded into the binary. While Jiff also embeds time zone into the binary when necessary (like on Windows), it doesn't expose that fact because it uses `/usr/share/zoneinfo` on Unix.